contracts/exchangeIssuance/ExchangeIssuance.sol

```diff
@@ -327,7 +327,8 @@ contract ExchangeIssuance is ReentrancyGuard {
     {
         require(_amountSetToRedeem > 0, "ExchangeIssuance: INVALID INPUTS");
         
-        uint256 amountEthOut = _redeemExactSetForWETH(_setToken, _amountSetToRedeem);
+        _redeemExactSetForWETH(_setToken, _amountSetToRedeem);
+        uint256 amountEthOut = IERC20(WETH).balanceOf(address(this));
```

Comment:
Why not use the `amountEthOut` returned from `_redeemExactSetForWETH()`, rather than using the current balance?

Reply:
The reason I changed this is that `_redeemExactSetForWeth` appears to calculate its return value incorrectly when there is a weth component in the set. I haven't been able to find the problem, so I just fetched the weth balance of the contract instead.